Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:18.04
+
+RUN useradd -u 1001 -ms /bin/bash dockeruser
+
+RUN set -x \
```

Comment:
Tox requires all versions of python to be installed. This dockerfile is purely for tests